### ISSUE TITLE
Characters at the bottom of a large initiative order are cut off by the encounter difficulty bar.

### ISSUE THREAD

**Issue by SonofOrion42** [astranauta/5etoolsR20#7]
Character Bael is seen at the top of the order here: https://pasteboard.co/GGV0eTp.png

There is another character named Errk directly below the second Fellforged Templar that is out of view here: https://pasteboard.co/GGV1eLM.png

Next character's turn: https://pasteboard.co/GGV20hg.png

View at bottom of turn order. Errk is now visible. Bael, who is directly after Errk, is not: https://pasteboard.co/GGV2zsu.png

The window is unable to scroll any further down. This will undoubtedly cause issues with pulling up sheets via the macro, updating HP in initiative window, etc., in the future.

**Comment by MrLabRat** [comment]
How many players + creatures do you have in there?

**Comment by SonofOrion42** [comment]
I had 16 characters in that initiative order. I did a little more experimenting afterwards, and I discovered that the initiative window works properly, with no cut-off, if everyone rolls initiative while I have the window open. However, if a lot of characters roll initiative before the window is opened, or I close and reopen the window any time after initiative is rolled, the bottom is cut out. This usually happens when around 9 or more characters are in the initiative order.
